examples/Console/TestPrometheusExporter.cs

```diff
@@ -52,7 +52,7 @@ internal static object Run(int port, int totalDurationInMins)
                 .AddPrometheusExporter(opt =>
                 {
                     opt.StartHttpListener = true;
-                    opt.HttpListenerPrefixes = new string[] { $"http://*:{port}/" };
+                    opt.HttpListenerPrefixes = new string[] { $"http://localhost:{port}/" };
```

Comment:
Interesting! Happen to catch an exception message or anything I can look into?

Reply:
![image](https://user-images.githubusercontent.com/5232798/135533910-7a52d084-1e12-4c4a-9509-271b201317b0.png)

Separately, we want to ensure no exception are thrown from the Worker task, as it is not observed anywhere.